programs/lib/include/flexpret_csrs.h

```diff
@@ -18,6 +17,8 @@
 #define CSR_TOHOST(tid) (CSR_TOHOST_BASE + tid)
 
 #define CSR_HWLOCK      0x520
+#define CSR_COMPARE_DU_WU 0x521
+#define CSR_COMPARE_IE_EE 0x522
```

Comment:
Before this PR, delay until/wait until and interrupt on expire/exception on expire shared a timer register. That made it impossible to use DU/WU at the same time as IE/EE. We should be able to use them together, however, because it is desired to be able to schedule an interrupt in the future and sleep wait/delay until that interrupt occurs.

There are more changes in the .scala files about this.

The old CSR comment should be removed though.